about.html

```diff
@@ -0,0 +1,153 @@
+  <link rel="stylesheet" href="style.css">
+  <link href="http://maxcdn.bootstrapcdn.com/font-awesome/4.1.0/css/font-awesome.min.css" rel="stylesheet">
+  <title>About page CAPSTONE</title>
+</head>
+<body class="bg-gray-light-8">
+  <nav class="first-nav bg-blue text-white">
+    <a href="https://www.facebook.com/"><i class="fa fa-facebook"></i></a>
+    <a href="https://twitter.com/home"><i class="fa fa-twitter"></i></a>
+    <ul>
+      <li>
+        Hungarian
+      </li>
+      <li>
+        My Profile
+      </li>
+      <li>
+        Logout
+      </li>
+    </ul>
+  </nav>
+  <nav class="navbar pb-1 bg-blue">
+    <ul class="nav-menu bg-blue">
+      <li class="nav-item">
+        <a href="./index.html" class="nav-link">Homepage</a>
+      </li>
+      <li class="nav-item">
+        <a href="#featured" class="nav-link">Featured Artists</a>
+      </li>
+      <li class="nav-item">
+        <a href="#" class="nav-link">Tickets</a>
+      </li>
+      <li class="nav-item">
+        <a href="#" class="nav-link">Info</a>
+      </li>
+    </ul>
+    <div class="hamburger">
+      <span class="bar"></span>
+      <span class="bar"></span>
+      <span class="bar"></span>
+    </div>
+    <div class="header-logo">
+      <a href="https://szigetfestival.com/en/"><img class="header-img" src="./images/sz-color-logo3-removebg-preview.png" alt="sziget logo"></a>
+    </div>
+    <ul class="menu-desktop">
+      <li><a href="./index.html">Homepage</a></li>
+      <li>Lineup</li>
+      <li>Tickets</li>
+      <li>Info</li>
+    </ul>
+  </nav>
+    <div id="about" class="about pb-2 pt-4">
+      <h2 class="h4-about pt-1 pb-2 text-red">green Sziget
+      </h2>
+      <h1 class="h1-about pb-2 text-red-dark-2">SZIGET Festival 2022</h1>
+      <div class="card p-1 m-1 col-10-md col-8-lg col-6-xl bg-white">
+        <p class="card-body">One of the Love Revolution’s main message is about environmental consciousness and sustainable development. 
+          Last year, we put a lot of effort into fulfilling our goals and becoming greener and plan on continuing with this tradition this year. 
+          The Love Revolution is our way of celebrating, that together, we can achieve sustainable development and environmental awareness in our daily lives.
+        </p>
+      </div>
+      <div class="contact-info p-1 mt-1 ml-2 mr-2 col-7-md col-5-lg col-4-xl bg-white"> 
+        <p>we are always here for you on the following email address: <br>
+          <a href="mailto: hello@szigetfestival.com">hello@szigetfestival.com</a> <br> 
+          If you have questions regarding ticketing purpose, please contact us here: <br> 
+          <a href="mailto: ticket@szigetfestival.com">ticket@szigetfestival.com</a>
+        </p>
+      </div>
+    </div>
+    <div class="sziget-logo p-2 mt-2 mb-4 bg-red">
+      <img src="./images/sziget-logo.jpeg" alt="sziget logo">
+    </div>
+    <div class="past-fests bg-white">
```

Comment:
- [ ] the about page is far a bit from the design guidelines kindly update the page and section to match the design as match as you can

your design

![image](https://user-images.githubusercontent.com/89282221/190517661-f2c4ba63-264f-46f1-a9e5-13ebde515c1d.png)

template design 
![image](https://user-images.githubusercontent.com/89282221/190517730-b93af7be-3e7e-49be-8543-927a33d81e51.png)

Reply:
I don't even understand what that means, it has different content, so it looks different

Reply:
@MattGomb the content can be different but the layout should stay the same , you can customize the text ,images the data of cards etc ... But you should use the same sizes, colors and the layout of the template , its in the requirements

Reply:
@ichala The layout is exactly the same, the sizes are as much as the same as it can still look decent, and the colors are the same